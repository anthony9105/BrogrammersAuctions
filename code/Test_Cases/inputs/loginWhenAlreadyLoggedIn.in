login adminUser1                  
login anotherUser                 