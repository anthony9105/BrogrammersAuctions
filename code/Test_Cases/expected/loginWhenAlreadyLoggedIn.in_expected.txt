Login successful
Error.  You are already logged in.